path_role.go

```diff
@@ -440,6 +447,10 @@ func (b *jwtAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical.
 		role.VerboseOIDCLogging = verboseOIDCLoggingRaw.(bool)
 	}
 
+	if maxAgeRaw, ok := data.GetOk("max_age"); ok {
+		role.MaxAge = time.Duration(maxAgeRaw.(int)) * time.Second
```

Comment:
What are your thoughts on erroring/warning if `max_age` is a negative number?

Reply:
The request validation for field data handles returning an error if `max_age` is a negative number (see [field_data.go#L219](https://github.com/hashicorp/vault/blob/master/sdk/framework/field_data.go#L219)). The error looks like: 
```
error converting input -2 for field "max_age": cannot provide negative value '-2'
```